src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -235,25 +290,56 @@ public function getConfigTreeBuilder()
                         ->end()
                     ->end()
                 ->end()
+            ->end();
+    }
 
+    private function addMercureSection(ArrayNodeDefinition $rootNode): void
+    {
+        $rootNode
+            ->children()
                 ->arrayNode('mercure')
                     ->{class_exists(MercureBundle::class) ? 'canBeDisabled' : 'canBeEnabled'}()
                     ->children()
                         ->scalarNode('hub_url')
                             ->defaultNull()
-                            ->info('The URL send in the Link HTTP header. If not set, will default to the URL for the Symfony\'s bundle default hub.')
+                            ->info('The URL sent in the Link HTTP header. If not set, will default to the URL for MercureBundle\'s default hub.')
                         ->end()
                     ->end()
                 ->end()
+            ->end();
+    }
 
+    private function addMessengerSection(ArrayNodeDefinition $rootNode): void
+    {
+        $rootNode
+            ->children()
                 ->arrayNode('messenger')
-                    ->{interface_exists(MessageBusInterface::class) ? 'canBeDisabled' : 'canBeEnabled'}()
+                    ->{!class_exists(FullStack::class) && interface_exists(MessageBusInterface::class) ? 'canBeDisabled' : 'canBeEnabled'}()
```

Comment:
Not sure to get why you check for the `FullStack` class, messenger would work without it no?

Reply:
It's consistent with what's done in the FrameworkBundle, so that we use the correct default.

https://github.com/symfony/symfony/blob/v4.2.4/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php#L1003